### PR TITLE
make collection description editable

### PR DESCRIPTION
**Pull request by lusol**
Can now edit descriptions of collections

screen shot 2018-07-31 at 11 05 15 am
screen shot 2018-08-03 at 10 36 54 am

### CODE REVIEW

**Comment by pranavrc** [comment]
tests?

**Comment by lusol** [comment]
true true true

File: app/components/collection-view/template.hbs [hunk 1]
```diff
@@ -6,7 +6,12 @@
     <div class="col-md-7">
       <h2 class="header__name">{{collection.name}}</h2>
         {{#if collection.description}}
-          <p class="header__description">{{collection.description}}</p>
+          {{#if editingDescription}}
+            <textarea class="edit-area">{{collection.description}}</textarea>
+            <button class="save btn btn-default" {{action 'saveDescription'}}>save</button>
```
Comment (tkyi) [inline app/components/collection-view/template.hbs]: Why all lowercase?
`save` => `Save`

File: app/components/collection-view/styles.scss [hunk 2]
```diff
@@ -10,6 +10,25 @@
...
+    padding-left: 5px;
+  }
+
+  .edit-area {
+    resize: none;
+    position: relative;
+    width: 100%;
+  }
+
+  .save {
```
Comment (tkyi) [inline app/components/collection-view/styles.scss]: can you also color the button blue

**Comment by tkyi** [comment]
Can you also make the text form have slightly rounded corners?

File: app/components/collection-view/template.hbs (hunk 1 above)
Comment (lusol) [reply app/components/collection-view/template.hbs]: done

File: app/components/collection-view/styles.scss (hunk 2 above)
Comment (lusol) [reply app/components/collection-view/styles.scss]: done

**Pull request merged**